src/styles/sidebar/components/annotation-body.scss

```diff
@@ -0,0 +1,38 @@
+@use '../../variables' as var;
+
+.annotation-body {
+  @include var.font-normal;
+  color: var.$grey-6;
+  // Margin between top of ascent of annotation body and
+  // bottom of ascent of annotation-quote should be ~15px.
+  margin-top: var.$layout-h-margin - 5px;
+  margin-bottom: var.$layout-h-margin;
+  margin-right: 0px;
+  margin-left: 0px;
+}
+
+.annotation-body__text {
+  // Hidden annotations displayed to moderators, where the content is still
+  // present.
+  &.is-hidden.has-content {
+    text-decoration: line-through;
+    filter: grayscale(100%) contrast(65%);
+  }
+
+  // Hidden annotations displayed to non-moderators, where the content has been
```

Comment:
Oooh. Learning opportunity! In what state of the app would we display this? My understanding—perhaps faulty!—was that we didn't display hidden annotations _at all_ to non-moderators. Is this shown to the annotation's author? Something else?

Reply:
I believe it is possible, or was possible at some point, for hidden annotations to be returned to non-moderators with their contents "redacted". I didn't check when I made these changes under what circumstances that can still happen on the "h" side. It might be for replies that have been moderated but which have not moderated replies:

```
[ Original annotation (unmoderated) ]
  |
  -- [ Reply (moderated ) ] <-- Shown as a redacted annotation (?)
        |
       [ Reply to reply (unmoderated) ]
```